package.json

```diff
@@ -26,7 +26,7 @@
     "typescript": "5.5.4"
   },
   "scripts": {
-    "postinstall": "yarn workspace @kaoto/camel-catalog run build",
+    "postinstall": "yarn workspace @kaoto/camel-catalog run build && yarn workspace @kaoto/xml-schema-ts run build",
```

Comment:
I think we can run the build process in both packages at the same time, but let's check it after this PR